docs/src/model_components/biogeochemical/NPZ.md

```diff
@@ -2,7 +2,7 @@
 
 The provided NPZD model is the low complexity model of [Kuhn2015](@citet).
 
-<img src="npzd.svg" width="700">
+![Diagram of NPZD formulation](npzd.svg)
```

Comment:
````suggestion
```@raw html
<img src="npzd.svg" width="700" alt="Diagram of NPZD formulation">
```
````

Maybe something like this?